assets/sass/utilities/_text-align.scss

```diff
@@ -0,0 +1,24 @@
+/**
+ * Text alignment utility styles.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-text-align-center {
+		text-align: center !important;
```

Comment:
😱 

I know sometimes when dealing with frameworks and giant libraries of SCSS we need to use `!important`, but I've always found it helpful to annotate _why_ it's needed/what's stopping us from omitting it. That makes refactoring later easier and gives context to the choice, rather than having them build up over time. What's the reason for this `!important` use? If you could add it in a comment here that'd be awesome 🙂

Reply:
😄 
It's simply a utility class, so it makes sense to use `!important` as it's intended to trump other rules and we can't rely on specificity alone since this selector has very little. We should always advocate for not using `important` in module or other non-utility styles but utilities are an exception I would say. The margin utility we have has this as well.

Reply:
I would advise against the `!important` flag unless we actually run into an issue with it not being specific enough, at which point we could reconsider. When looking at our usage of it, it appears to be without it would work just as well, so let's remove the flag.